scripts/set-versions.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+command -v shellcheck >/dev/null && shellcheck "$0"
+
+readonly ADMIN_MODULE_DIR="admin-module-src"
+readonly BLOCK_SDK_DIR="block-sdk-src"
+readonly COSMOS_SDK_DIR="cosmos-sdk-src"
+readonly FEEMARKET_DIR="feemarket-src"
+readonly IBC_GO_DIR="ibc-go-src"
+readonly NEUTRON_DIR="neutron-src"
+readonly SLINKY_DIR="slinky-src"
+readonly WASMD_DIR="wasmd-src"
+
+readonly ADMIN_MODULE_REV="v2.0.2"
+readonly BLOCK_SDK_REV="v2.1.5"
+readonly COSMOS_SDK_REV="v0.50.9-neutron"
+readonly FEEMARKET_REV="v1.1.1"
+readonly IBC_GO_REV="v8.5.1"
+readonly NEUTRON_REV="v4.2.4"
```

Comment:
My bad, should be main. Regenerate all files as well

Reply:
Done